src/ripple/rpc/handlers/LedgerHandler.cpp

```diff
@@ -212,13 +213,105 @@ doLedgerGrpc(RPC::GRPCContext<org::xrpl::rpc::v1::GetLedgerRequest>& context)
                 obj->set_mod_type(org::xrpl::rpc::v1::RawLedgerObject::DELETED);
             else
                 obj->set_mod_type(org::xrpl::rpc::v1::RawLedgerObject::CREATED);
+            auto blob = inDesired ? inDesired->slice() : inBase->slice();
+            LedgerEntryType objectType =
+                LedgerEntryType{(uint16_t)(blob[1] << 8 | blob[2])};
```

Comment:
I tend to avoid c-style casts. Can't we just cast directly to the `LedgerEntryType`? I.e.
```
            auto const objectType =
                static_cast<LedgerEntryType>(blob[1] << 8 | blob[2]);
```
Also, I think we can make this `const`.